bucket/mkvtoolnix.json

```diff
@@ -1,19 +1,34 @@
 {
     "version": "39.0.0",
-    "homepage": "https://mkvtoolnix.download/",
+    "homepage": "https://mkvtoolnix.download",
     "description": "A set of tools to create, alter and inspect Matroska files.",
     "license": "GPL-2.0-only",
     "architecture": {
         "64bit": {
-            "url": "https://mkvtoolnix.download/windows/releases/39.0.0/mkvtoolnix-64-bit-39.0.0.7z",
+            "url": "https://www.fosshub.com/MKVToolNix.html/mkvtoolnix-64-bit-39.0.0.7z",
             "hash": "sha512:b97767fdb1653f24f1b7e3c1b857cfb8237edebffcd7d12714098a6f0348ea002e6516fbd76821575eff767393ad13f1ea4f82284bfd3a150ea4c560d9ffa89c"
         },
         "32bit": {
-            "url": "https://mkvtoolnix.download/windows/releases/39.0.0/mkvtoolnix-32-bit-39.0.0.7z",
+            "url": "https://www.fosshub.com/MKVToolNix.html/mkvtoolnix-32-bit-39.0.0.7z",
             "hash": "sha512:69be8e72ab2f80a55b82b2b81921e85c9accda098966f14762258b073f559a6662b4fe10f2b22f54516465de4565faa831aaf1a81a0b8e0161717803c81a706b"
         }
     },
     "extract_dir": "mkvtoolnix",
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\mkvtoolnix-gui.ini\")) {",
+        "    Set-content \"$dir\\mkvtoolnix-gui.ini\" -Value @('[settings]', 'updates\\checkForUpdates=false') -Encoding Ascii",
+        "} else {",
+        "# Persist mkvtoolnix-gui.ini manully because symlink doesn't work.",
+        "    Copy-Item \"$persist_dir\\mkvtoolnix-gui.ini\" \"$dir\"",
+        "}"
+    ],
+    "uninstaller": {
+        "script": [
+            "if (Test-Path \"$persist_dir\\mkvtoolnix-gui.ini\") {",
```

Comment:
Why do you check if this file exists?

Reply:
Sorry, a mistake.

Reply:
Still exists

Reply:
I check if it exists before copy it.

Reply:
And why do you do it?

Reply:
If the file doesn't exist copy will fail.

Reply:
Now it is correct with dir. Previous persist_dir was nonsence

Reply:
Yeah, it's a mistake...